proxy_test.go

```diff
@@ -213,7 +246,7 @@ func (suite *ProxyTests) TestServeHTTPS3Unresponsive() {
 func (suite *ProxyTests) TestServeHTTP_allowed() {
 
 	// Start proxy that allows everything
-	proxy := NewProxy(suite.S3conf, NewAlwaysAllow(), suite.messenger, new(tls.Config))
+	proxy := NewProxy(suite.S3conf, NewAlwaysAllow(), suite.messenger, suite.database, new(tls.Config))
```

Comment:
```suggestion
	database, err := common.NewSDAdb(suite.DBConf)
	if err != nil {
		suite.T().Skip("skip TestShutdown since broker not present")
	}
	proxy := NewProxy(suite.S3conf, NewAlwaysAllow(), suite.messenger, database, new(tls.Config))
```